### PR TITLE
overload imread

### PR DESCRIPTION
**Pull request by sturkmen72**
https://github.com/Itseez/opencv/pull/4201 i need some advices

### CODE REVIEW

File: modules/imgcodecs/include/opencv2/imgcodecs.hpp [hunk 1]
```diff
@@ -131,13 +131,7 @@ returns an empty matrix ( Mat::data==NULL ). Currently, the following file forma
  */
 CV_EXPORTS_W Mat imread( const String& filename, int flags = IMREAD_COLOR );
 
-/** @brief Loads and resizes down an image from a file.
-@anchor imread_reduced
-@param filename Name of file to be loaded.
-@param flags Flag that can take values of @ref cv::ImreadModes
-@param scale_denom
- */
```
Comment (alalek) [inline modules/imgcodecs/include/opencv2/imgcodecs.hpp]: Documentation should be added into header files (remove documentation comments from `.cpp` file)
Also use `@overload` doxygen command.

File: modules/imgcodecs/src/grfmt_jpeg.cpp [hunk 2]
```diff
@@ -242,9 +242,10 @@ bool  JpegDecoder::readHeader()
         {
             jpeg_read_header( &state->cinfo, TRUE );
 
+            int m_scale_denom=m_size.width>8 ? 1 : m_size.width;
```
Comment (sturkmen72) [inline modules/imgcodecs/src/grfmt_jpeg.cpp]: i will add some calculations here

**Comment by sturkmen72** [comment]
i am working to achive

**Comment by sturkmen72** [comment]
https://github.com/Itseez/opencv/pull/5083

**Comment by PhilLab** [comment]
@sturkmen72 FYI: If you force push (``--force``) to the same branch, you don't have to open multiple pull requests

**Comment by sturkmen72** [comment]
@PhilLab thank you for your comment. i know that. but there was a problem on my local GitHub. when i tried to squash the branch sturkmen72:patch-3 i can't access patch-3

**Pull request closed without merging**